lib/cogctl/actions/users/info.ex

```diff
@@ -1,41 +1,50 @@
 defmodule Cogctl.Actions.Users.Info do
   use Cogctl.Action, "users info"
-  alias Cogctl.Table
+  import Cogctl.Actions.Users.Util, only: [render: 4]
 
   def option_spec do
     [{:user, :undefined, :undefined, {:string, :undefined}, 'User username (required)'}]
   end
 
+  def run(options, _args, _config, %{token: nil}=endpoint),
+    do: with_authentication(endpoint, &run(options, nil, nil, &1))
   def run(options, _args, _config, endpoint) do
-    with_authentication(endpoint,
-                        &do_info(&1, :proplists.get_value(:user, options)))
-  end
-
-  defp do_info(_endpoint, :undefined) do
-    display_arguments_error
+    case :proplists.get_value(:user, options) do
+      :undefined ->
+        display_arguments_error
```

Comment:
I think you'll want to update this to take advantage of changes @mpeck made re: error messages.